provision/roles/mysql/tasks/main.yml

```diff
@@ -9,18 +9,36 @@
     - python-selinux
     - python-mysqldb
 
-- mysql_user: name=root host={{ item }} password={{ mysql_root_password }} priv=*.*:ALL,GRANT
+- name: Set root password
+  mysql_user:
+    name: root
+    host: '{{ item }}'
+    password: '{{ mysql_root_password }}'
   with_items:
     - "%"
```

Comment:
you might want to remove this one so that you can't just connect from the outside :wink: